pkg/operatorsource/handler.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 
 	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/shared"
-	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	v1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
```

Comment:
What's the point of this alias?

Reply:
That is weird. I have an after-save hook to tell `gopls` to organize imports, but I didn't notice that it had done that.